### PR TITLE
Make internal links consistent

### PR DESCRIPTION
**Pull request by linuxdaemon**

### CODE REVIEW

**Comment by n7st** [comment]
I think ideally the content should be adjusted to use proper markdown permalinks i.e. [whatever](/whatever) so links work across all environments (dev/staging/prod).

**Review by n7st — CHANGES_REQUESTED** [review]
I think ideally the content should be adjusted to use proper markdown permalinks i.e. `[whatever](/whatever)` so links work across all environments (dev/staging/prod).

File: content/community/anxiety/mods.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ type: page
...
 
-* Use your discretion when dealing with problematic users. Sometimes a simple topic change could handle the situation. Sometimes a user may deserve a warning. Sometimes a user will not relent until they are banned. You may choose to PM a user about their behavior or maybe just gently prod them in channel. However you choose to do it, a warning is preferred before any more severe action. Try referring people to the link to #anxiety rules in the topic. (www.snoonet.org/anxietyrules) Typically, the process should be:
+* Use your discretion when dealing with problematic users. Sometimes a simple topic change could handle the situation. Sometimes a user may deserve a warning. Sometimes a user will not relent until they are banned. You may choose to PM a user about their behavior or maybe just gently prod them in channel. However you choose to do it, a warning is preferred before any more severe action. Try referring people to the link to #anxiety rules in the topic. (snoonet.org/anxietyrules) Typically, the process should be:
```
Comment (n7st) [inline content/community/anxiety/mods.md]:
```
Try referring people to the link to #anxiety rules [in the topic](/anxietyrules). Typically, the process should be:
```

File: content/community/casualconversation.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ subreddit: "CasualConversation"
 
 **What sets your community apart from others?**
 
-* {{< subreddit_link CasualConversation >}} is a subreddit for general chat about *almost* anything that you can think of. The main rules here are to "respect others" and "Don't take things too seriously." Mostly we just want people to have a friendly place to have conversations but we do have a [detailed rules page here](http://www.reddit.com/r/CasualConversation/wiki/rules). Oh we also have a Snoonet rules page that can be found at https://www.snoonet.org/ccrules.
+* {{< subreddit_link CasualConversation >}} is a subreddit for general chat about *almost* anything that you can think of. The main rules here are to "respect others" and "Don't take things too seriously." Mostly we just want people to have a friendly place to have conversations but we do have a [detailed rules page here](http://www.reddit.com/r/CasualConversation/wiki/rules). Oh we also have a Snoonet rules page that can be found at https://snoonet.org/ccrules.
```
Comment (n7st) [inline content/community/casualconversation.md]:
```
Oh we also have a Snoonet rules page that can be found [here](/ccrules).
```

Comment (linuxdaemon) [reply content/community/casualconversation.md]: Fixed in aae7a23f41e58728cdb2080121a3d8b2effc4c28

File: content/community/anxiety/mods.md (hunk 1 above)
Comment (linuxdaemon) [reply content/community/anxiety/mods.md]: Fixed in aae7a23f41e58728cdb2080121a3d8b2effc4c28

**Review by n7st — APPROVED** [review]

**Pull request merged**